requirements.txt

```
@@ -24,30 +24,30 @@ mdurl==0.1.2
 orjson==3.10.18
 pillow==11.2.1
 pycparser==2.22
-pydantic==2.11.4
-pydantic-extra-types==2.10.4
+pydantic==2.11.6
+pydantic-extra-types==2.10.5
 pydantic-settings==2.9.1
 pydantic_core==2.33.2
 Pygments==2.19.1
 PyJWT==2.10.1
-pymongo==4.12.1
+pymongo==4.13.1
 python-dateutil==2.9.0.post0
 python-dotenv==1.1.0
 python-multipart==0.0.20
 PyYAML==6.0.2
 rich==14.0.0
-rich-toolkit==0.14.5
-s3transfer==0.10.4
+rich-toolkit==0.14.7
+s3transfer==0.13.0
```

The End-to-End Tests job is failing because of the issues recorded [here](http://github.com/ral-facilities/object-storage-api/actions/runs/15686747493/job/44191807370?pr=175#step:6:18410).
```suggestion
s3transfer==0.10.4
```